lib/usage-tracking/class-usage-tracking-base.php

```diff
@@ -430,6 +430,14 @@ protected function hide_tracking_opt_in() {
 	 * @return bool true if the opt-in is hidden, false otherwise.
 	 **/
 	protected function is_opt_in_hidden() {
+		$delayed_notice_timestamp  = (int) get_option( 'job_manager_display_usage_tracking_once' );
+
+		// Display only once they delayed notice regardless if the user has declined in the past.
+		if ( $delayed_notice_timestamp > 0 && $delayed_notice_timestamp < time() ) {
+			update_option('job_manager_display_usage_tracking_once', 0 );
```

Comment:
Usually WP complains about missing spaces on function calls like this.

I'm not sure why it didn't complain here, but I'm adding the suggestion anyway:

```suggestion
			update_option( 'job_manager_display_usage_tracking_once', 0 );
```

Reply:
It seems that we weren't running phpcs in the lib directory (maybe initially it was meant to only put libraries in there?). I added this dir, run phpcbf, added a couple of rules and updated existing code.